### PR TITLE
consolidate newrelic into single nri-bundle and migrate to terraform

### PR DESCRIPTION
**Pull request by joshrwolf**
This consolidates the following images into a single group:

- "newrelic-infrastructure-bundle"
- "newrelic-fluent-bit-output"
- "newrelic-k8s-events-forwarder"
- "newrelic-prometheus-configurator"
- "nri-kube-events" -> "newrelic-nri-kube-events"
- "nri-kubernetes" -> "newrelic-nri-kubernetes"
- "nri-prometheus" -> "newrelic-nri-prometheus"

this cuts our fanout for this group of images by 6

### CODE REVIEW

File: images/newrelic/tests/main.tf [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  })
+}
+
+data "oci_string" "ref" {
+  for_each = var.digests
+  input    = each.value
+}
+
+variable "newrelic-license-key" {
+  default = "72ce21fc3aceba38b421229d2fa95caf1e9fNRAL"
```
Comment (joshrwolf) [inline images/newrelic/tests/main.tf:27]: gah, I need to fix this

File: images/newrelic/tests/main.tf [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+data "oci_string" "ref" {
+  for_each = var.digests
+  input    = each.value
+}
+
+# This is a pretty cursed method of getting the license key directly from the
+# environment, without needing to plumb through TF_VAR_
+data "external" "license-key" {
+  program = ["bash", "-c", "echo \"{\\\"output\\\": \\\"$NEW_RELIC_LICENSE_KEY\\\"}\""]
+}
```
Comment (imjasonh) [inline images/newrelic/tests/main.tf:30]: Wow. I'm not even mad, I'm impressed.

We should definitely drop this, but in the meantime you've taught me a new cursed way to pipe things through, and I'll never forgive you. 😁

File: images/newrelic/tests/main.tf [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+global:
+  cluster: test
+  licenseKey: ${data.external.license-key.result.output}
+
+newrelic-infrastructure:
+  images:
+    # forwarder:
+    #   registry:
+    #   repository:
+    #   tag:
```
Comment (imjasonh) [inline images/newrelic/tests/main.tf:49]: Is this not needed because `nri-kube-events.images` does this below?

Comment (joshrwolf) [reply images/newrelic/tests/main.tf]: fixed! good catch

**Review by imjasonh — APPROVED** [review]

**Pull request merged**